tests/example_data/raw/split.py

```diff
@@ -0,0 +1,39 @@
+"""
+This is a small script to split the raw data into different subsets to be used while testing.
+
+The data is the 151 first pokemon and the following field are available:
```

Comment:
Not a fan of adding Parquet files to the Github repository, and as they stay in the commit history they stay there forever so just removing them later won't make it faster.

I'd host the Parquet files on the hub and load them using the `hf_hub_download` [method](https://huggingface.co/docs/huggingface_hub/guides/download)

Reply:
As long as we stay in the range of kilobytes, I don't see an issue with this. Adding data files for the tests is fine as long as the goal is to validate format handling and they contain minimal data.

Moving the files out of the repo decouples them from the versioning and adds complexity.

Reply:
Moving the files out of the repo also makes our tests dependent on an internet connection, which will only bring pain.

Some best practices for tests:
https://web.archive.org/web/20210510024513/http://beyondcoding.net/articles/firstprinciples.html